governance/polls/DAO Masons Poll Exercise 2 - README.md

```diff
@@ -0,0 +1,115 @@
+# DAO Masons - Poll Exercise 2
+
+Determine what course of action you would take as a Governance Facilitator and communicate it clearly.
+
+1. We know the information provided by Diane provides 'reasonable suspicion' that Clive's identity has been revealed. I'm assuming that 'reasonable suspicion' is equivalent to the 'significant suspicion' threshold referred to in [Governance Scope 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) that requires a FacilitatorDAO to immediately derecognize the AD.
+
+2. According to [Governance Scope 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) I should 'immediately derecognize the AD' and could be penalized if I don't.
+
+   The Governance Scope also assumes the existence of multiple FacilitatorDAOs who can review my decision. Since those don't currently exist, I think a good compromise is to go ahead with the poll and let Maker holders decide.
+
+3. I would make the following forum response to the original thread:
+
+> [Governance Scope 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) defines a policy that requires me in my role as Governance Facilitator to take action when there is 'significant suspicion' that an AD's identity has been revealed. I believe what Diane has presented represents a 'significant suspicion' and so as a Facilitator I am bound to act.
+
+> According to [Governance Scope 2.6.5.1](https://mips.makerdao.com/mips/details/MIP101#2-6-5-1), FaciliatorDAO could immediately derecognize Clive as an AD. Due to the current lack of official FacilitatorDAOs described in governance to provide checks and balances, I've deemed it is more appropriate for Maker holders to make the determination in this case. In the future this decision may be made by FacilitatorDAOs unilaterally.
+
+> I've reached out to Clive to give him a chance to make a rebuttal, and will include links to his comments in the upcoming Poll. The poll is scheduled to begin on Monday July 31st.
+
+4. I would simultaneously reach out to Clive and let him know why I started a governance poll and ask him if he'd like to make a statement to be included in the poll.
```

Comment:
Yeah, points for this. I don't think anyone else came up with: 'I'll go talk to Clive' as one of their responses. Feels pretty important given the materials provided.